closed/OpenJ9.gmk

```diff
@@ -365,7 +365,13 @@ $(J9JCL_SOURCES_DONEFILE) : \
 	@$(ECHO) Generating J9JCL sources
 	$(call RunJPP, GENERIC, $(TOPDIR)/closed/adds/jdk/src/share/classes, $(JPP_DEST)) \
 		$(IncludeIfUnsure)
+	$(call RunJPP, SIDECAR18-SE-$(if $(CLOSEDJ9_SHA),CLOSED,OPEN)J9, $(OPENJ9_TOPDIR)/jcl, $(JPP_DEST))
```

Comment:
Line 368 is now redundant; please remove it.

Reply:
Oops, updated.